lib/internal/fs/watch/linux.js

```diff
@@ -0,0 +1,244 @@
+'use strict';
+
+const { SafeMap, Symbol, StringPrototypeStartsWith, SymbolAsyncIterator, Promise } = primordials;
```

Comment:
For consistency with other files, let's use a multiline destructuring syntax, and let's order the properties in alphabetical order:
```suggestion
const {
  Promise,
  SafeMap,
  StringPrototypeStartsWith,
  Symbol,
  SymbolAsyncIterator,
} = primordials;
```

Reply:
Thank you! Can we add an eslint rule for this? Or do we have a note on the contributing docs regarding this? I don't want to keep forgetting this.

Reply:
I'd love to have a ESLint for that, if you want to introduce such thing, that'd be awesome (but probably not in this PR).

Reply:
Thank you for the feedback. I fixed this requested changes, and will follow up with a different pull request for this eslint rule.